Dashboard/Connectors/Serial/SerialConnector.cs

```diff
@@ -143,6 +175,42 @@ private void ParseResMessage(Dictionary<string, string> values)
         });
     }
 
+    private DateTime ParseUTCTime(string timeString)
+    {
+
+        string datePart = "";
```

Comment:
It would be nice to have it already be in a standard format. We can bring it up to Nick O next meeting if we have time.

Reply:
It would be good to have it already in a standard format, We could bring it up to Nick O but given the timeline we currently have, its not a big issue.